src/DXRModules/DeusEx/Classes/DXRHints.uc

```diff
@@ -249,7 +249,11 @@ simulated function InitHints()
         }
         if(dxr.flags.settings.min_weapon_dmg != dxr.flags.settings.max_weapon_dmg || dxr.flags.settings.min_weapon_shottime != dxr.flags.settings.max_weapon_shottime) {
             AddHint("Each type of weapon gets randomized stats!", "Make sure to check one of each type.");
-            AddHint("Each type of weapon will have the same stats", "through the whole game.  They won't change later!");
+            if (dxr.flags.IsWaltonWare() || (dxr.flags.settings.bingo_win > 0 && class'MenuChoice_NewGamePlus'.default.value == 2)) {
```

Comment:
maybe should also check for MBM mode since that also uses bingo_win>0 but with a different meaning

or maybe we should have a getter function for bingo_win vs bingo_requirement lol